__tests__/product/product.mutation.test.js

```diff
@@ -125,61 +130,51 @@ describe('Product mutations', () => {
     const createdProduct = createProduct.data.addProduct;
     expect(createdProduct).toBeDefined();
     expect(createdProduct).toHaveProperty('name', [
-      { __typename: 'Language', value: 'Very Coool Baggy' },
-      { __typename: 'Language', value: 'ДУЖЕ СУПЕРСЬКИЙ Рюкзачечок' },
+      { value: 'Very Coool Baggy' },
+      { value: 'ДУЖЕ СУПЕРСЬКИЙ Рюкзачечок' },
     ]);
     expect(createdProduct).toHaveProperty('description', [
-      { __typename: 'Language', value: 'Baggy is so cool' },
-      { __typename: 'Language', value: 'Рюкзачечок - супер кльовий))' },
+      { value: 'Baggy is so cool' },
+      { value: 'Рюкзачечок - супер кльовий))' },
     ]);
     expect(createdProduct).toHaveProperty('category', {
-      __typename: 'Category',
       _id: categoryId,
     });
     expect(createdProduct).toHaveProperty('subcategory', {
-      __typename: 'Category',
       _id: categoryId,
     });
     expect(createdProduct).toHaveProperty('mainMaterial', [
       {
-        __typename: 'Language',
         value: 'Canvas-400G прошита додатковим шаром спеціального матеріалу',
```

Comment:
Please, don't hardcode values which you assert, instead use properties of the object that you have used for the creation of the product. It will reduce the required changes if some data of the product is going to be changed

Reply:
solved